src/main.py

```diff
@@ -7,6 +7,8 @@
 www.yetitool.com
 '''
 from asmcnc import paths
+from asmcnc.comms.user_settings_manager import UserSettingsManager
+
 paths.create_paths()
```

Comment:
just in case, keep create_paths() above everything else in this file, just incase user settings manager had to use sb_values or other non-permanent path